### PR TITLE
[R4R] add issue and propose tutorial

### PR DESCRIPTION
**Pull request by yutianwu**
Add issue and propose tutorial scripts.

### CODE REVIEW

File: tutorial/issue_tutorial.sh [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+        continue
+    else
+        symbol=$(echo ${symbol} | tr a-z A-Z)
+        break
+    fi
+done
+
+# Get supply
+while true; do
+    echo "What is the total supply(real number)?"
```
Comment (huangsuyu) [inline tutorial/issue_tutorial.sh]: maybe use: actual total supply

File: tutorial/issue_tutorial.sh [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        continue
+    fi
+
+    if [[ $((supply)) -lt 1000 || $((supply)) -gt 90000000000 ]]
+    then
+        echo "supply should larger than 1000 and less than 90000000000"
+    fi
+
+    english_version=$(number2words ${supply})
+    echo "total supply is ${english_version}(${supply}), which will be showed as ${supply}00000000 in command, enter Y to continue, enter other key to input again"
```
Comment (huangsuyu) [inline tutorial/issue_tutorial.sh]: modify again

File: tutorial/issue_tutorial.sh [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+        echo "token name should only contains less than 33 alphanumeric characters and space, please try again"
+        continue
+    else
+        break
+    fi
+done
+
+# Get symbol
+while true; do
+    echo "What is the symbol?"
```
Comment (huangsuyu) [inline tutorial/issue_tutorial.sh]: maybe add the constrains: symbol should only contains 3~8 alphanumeric characters

File: tutorial/issue_tutorial.sh [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+    then
+        echo "${account_name} does not exist"
+        continue
+    fi
+    break
+done
+
+# Get token name
+while true; do
+    echo "What is the token name?"
```
Comment (huangsuyu) [inline tutorial/issue_tutorial.sh]: maybe add hint for constrains

File: tutorial/propose_tutorial.sh [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+        [Yy] )
+            break;;
+        * )
+            continue;;
+    esac
+done
+
+# Get base asset
+while true; do
+    echo "What is the base asset?"
```
Comment (huangsuyu) [inline tutorial/propose_tutorial.sh]: maybe we need to add a query for the asset symbol

**Comment by darren-liu** [comment]
Instead of "tutorial", the scripts are more like handy tools. I suggest you change path to "tools".

**Comment by yutianwu** [comment]
Updated

**Pull request merged**